components/Theme.tsx

```diff
@@ -0,0 +1,19 @@
+import {useEffect, useState} from "react";
```

Comment:
Codacy has a fix for the issue: [Replace `useEffect,·useState` with `·useEffect,·useState·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
import { useEffect, useState } from "react";
```